coralproject/talk
# Edit Comment Window Length Setting

## What does this PR do?

Implements story [EDIT SETTINGS | As an admin, I want to set a timeframe for commenters to edit their comment](https://www.pivotaltracker.com/story/show/144459753)

## How do I test this PR?

* Visit /admin/configure as an admin
* Use left nav to go to "Moderation Settings" (we should really make this permalinkable)
* Scroll to bottom, see ![](http://i.imgur.com/jDELtdF.png)
* Change the edit comment window
* Click "Save Changes" (you may have to scroll up)
* Visit comment stream, leave a comment
* Click to edit your comment.
* Ensure that the edit countdown timer is counting down from the number of seconds you just configured
